xendit-android/src/main/java/com/xendit/Xendit.java

```diff
@@ -265,10 +265,10 @@ public String getCardKey() {
      * @param tokenCallback The callback that will be called when the token creation completes or
      *                      fails
      */
-    public void createSingleUseToken(final Card card, final int amount, final TokenCallback tokenCallback) {
+    public void createSingleUseToken(final Card card, final int amount, final boolean shouldAuthenticate, final TokenCallback tokenCallback) {
```

Comment:
This is a breaking change. We should either create a new method signature and @deprecate the old one if we don't want anyone to use it, or support both. I think supporting both is fine, the old one can default to true